calico_node/tests/st/bgp/test_bgp.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2018-2019 Tigera, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+from subprocess import CalledProcessError
+
+from tests.st.test_base import TestBase
+from tests.st.utils.docker_host import DockerHost, CLUSTER_STORE_DOCKER_OPTIONS
+from tests.st.utils.utils import check_bird_status, retry_until_success
+
+_log = logging.getLogger(__name__)
+_log.setLevel(logging.DEBUG)
+
+
+class TestReadiness(TestBase):
```

Comment:
I think we should at least have one more test where BGP is disabled (CALICO_NETWORKING_BACKEND=none), Felix readiness enabled.

It would be nice to have tests which check the other readiness conditions as well - e.g. graceful restart in progress